starter-code/index.html

```diff
@@ -270,7 +270,7 @@ <h2>Your pizza's price</h2>
 
       <section class="crust crust-gluten-free">
         <section class="cheese"></section>
-        <section class="sauce sauce-white"></section>
```

Comment:
Because you removed the 'sauce-white' class from here, your active button on the JS doesn't work as it should. Your white sauce button is active when there is tomato sauce. By adding the class back, everything works as it should :)